libr/core/cmd_anal.c

```diff
@@ -4945,8 +4945,12 @@ static int compute_coverage(RCore *core) {
 	RListIter *iter;
 	RAnalFunction *fcn;
 	int cov = 0;
+	ut64 section_begin = r_num_get (core->num, "$S");
```

Comment:
ok so as suggested by @alvarofe we should then consider only the sections with `x` bit to calculate the coverage.